### PR TITLE
post features

### PR DESCRIPTION
**Pull request by preethasri**

### CODE REVIEW

**Review by msdeshmukh009 — APPROVED** [review]
The code looks fine.
Good to merge.

File: src/Pages/HomePage/HomePage.css [hunk 1]
```diff
@@ -0,0 +1,7 @@
+.home-container{display: flex;
+height:100vh;
+max-width: 1300px;
+margin-left: auto;
+margin-right: auto;
+padding:0 10px;
```
Comment (Chakravarthi0) [inline src/Pages/HomePage/HomePage.css:6]: suggestion: you can use rem here

File: src/app/store.js [hunk 2]
```diff
@@ -1,10 +1,13 @@
 import { configureStore } from '@reduxjs/toolkit'
 import authReducer from '../Pages/Authentication'
-
+import postSlice from '../features/post/postSlice'
+import userSlice from '../features/user/userSlice'
```
Comment (Chakravarthi0) [inline src/app/store.js:4]: import naming inconsistent

**Review by Chakravarthi0 — APPROVED** [review]
app is working as expected

**Pull request merged**